### DISCUSSION TITLE
Supported file types

### DISCUSSION THREAD

**Discussion by stealthrabbi** [katspaugh/wavesurfer.js#2703] [Q&A]
What are the supported file types for wavesurfer? I've seen it work with wav, mp3, ogg. But, it doesn't seem to like .nist files. I do not see anything in the documentation.

**Comment by katspaugh** [comment]
wavesurfer.js supports the file types for which your browser has codecs. It uses the native decodeAudioData.

**Comment by prakharpbuf** [comment]
Tried as many file types as I could on my` Chrome Version 110.0.5481.180 (Official Build) (64-bit)`

Worked: 
1. m4a
2. mp3
3. mp4
4. wav
5. aac
6. wma
7. flac
8. opus
9. ogg


Did not work:
1. ac3
2. aiff
3. voc
4. amr
5. ra
6. midi 
7. nist
